### PR TITLE
Implement and deploy Aragon 0.5 Beta templates

### PR DESCRIPTION
**Pull request by izqui**
See [readme](https://github.com/aragon/aragon-apps/blob/beta-template/templates/beta/readme.md)

### CODE REVIEW

File: templates/beta/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "scripts": {
+    "deploy:rinkeby": "truffle migrate --network rinkeby --reset",
+    "deploy:rpc": "truffle migrate --network rpc --reset"
+  },
+  "author": "Aragon Institution MTU <contact@aragon.one>",
+  "license": "GPL-3.0",
+  "devDependencies": {
+    "@aragon/id": "^1.0.0",
+    "eth-ens-namehash": "^2.0.8",
+    "truffle": "^4.0.5",
```
Comment (sohkai) [inline templates/beta/package.json]: Pin this to 4.0.5

File: templates/beta/migrations/2_deploy.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+const daoFactoryMigration = require('@aragon/os/migrations/3_factory')
+const MiniMeTokenFactory = artifacts.require('@aragon/os/contracts/lib/minime/MiniMeTokenFactory')
+const ENS = artifacts.require('@aragon/os/contracts/lib/ens/ENS.sol')
+const EtherToken = artifacts.require('@aragon/os/contracts/common/EtherToken.sol')
+
+const templates = ['DemocracyTemplate', 'MultisigTemplate']
+
+// ensure alphabetic order
+const apps = ['finance', 'token-manager', 'vault', 'voting']
```
Comment (sohkai) [inline templates/beta/migrations/2_deploy.js]: Maybe do a `.sort()` to ensure it?

File: templates/beta/migrations/2_deploy.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+const appIds = apps.map(app => namehash(require(`@aragon/apps-${app}/arapp`).appName))
```
Comment (sohkai) [inline templates/beta/migrations/2_deploy.js]: Make the `.json` explicit here

File: templates/beta/migrations/2_deploy.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  return x.map(c => c.address)
+}
+
+const newRepo = async (apm, name, acc, contract) => {
+  const c = await artifacts.require(contract).new()
+  return await apm.newRepoWithVersion(name, acc, [1, 0, 0], c.address, '0x1245')
+}
+
+module.exports = async (deployer, network, accounts) => {
+  let indexObj = require('../index.js')
```
Comment (sohkai) [inline templates/beta/migrations/2_deploy.js]: This should just be `'../../index.js'`... right?

**Comment by sohkai** [comment]
Note to self: we should also log the installed apps here (see https://github.com/aragon/aragon-apps/pull/95).

**Comment by luisivan** [comment]
I <3 how easy it is to create templates if you know what you're doing

**Pull request merged**